amp/WEB-INF/src/org/digijava/module/aim/action/AuditLoggerManager.java

```diff
@@ -61,11 +65,30 @@ public ActionForward modePrepare(ActionMapping mapping, ActionForm form,
             }
                 
         }
+
         Collection<AmpAuditLogger> logs=AuditLoggerUtil.getLogObjects(vForm.isWithLogin());
         
+        if (vForm.getSelectedUser() != null || vForm.getFilteredTeam() !=null) {
```

Comment:
if we are going to only allow filtering by team or by user i would simply the code by cleaning the value in the view so only team or user is selected